modules/source-bookshelf/lib/base_methods.js

```diff
@@ -1,3 +1,5 @@
+const bluebird = require('bluebird');
```

Comment:
Would you be cool with using bPromise here? I feel like that is a little more clear when reading the code.